src/app/styles.tsx

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  position: relative;
+  width: 100%;
+  min-height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  display: flex;
+  flex-direction: column;
+  overflow-y: hidden;
```

Comment:
q: why do we need an `overflow-y: hidden` here? If it's for mobile view, it may be better to address the size of content inside the div rather than cutting it off. If that's not possible, it seems better to me to just let it overflow instead of cutting off the content entirely.

Reply:
If we remove the `overflow-y`, the content will overlap with other sections of the home page which can make it look pretty messy. However, even now it behaves a little weird because everything within `TextContainer` will simply just hide behind the `StatisticsSection` with `overflow-y: hidden`.

Reply:
Yes, it's better not to arbitrarily overflow it, and instead account for the weird format itself - that's the root of the issue.